src/app/core/common-components/entities-table/entities-table.component.ts

```diff
@@ -339,7 +338,7 @@ export class EntitiesTableComponent<T extends Entity> implements AfterViewInit {
   private disableSortingHeaderForAdvancedFields(c: FormFieldConfig) {
     // if no dataType is defined, these are dynamic, display-only components
     if (
-      c.dataType === EntityArrayDatatype.dataType ||
+      // TODO: prevent sort for array data?
```

Comment:
Does not make so much sense to sort by a bunch of values. I would prevent the sort here.